warrant/__init__.py

```diff
@@ -346,6 +346,18 @@ def confirm_sign_up(self,confirmation_code,username=None):
         self._add_secret_hash(params, 'SecretHash')
         self.client.confirm_sign_up(**params)
 
+    def resend_confirmation_code(self,username=None):
```

Comment:
It is better with a whitespace after the comma.

Reply:
Agree and updated, thanks!